package.json

```diff
@@ -18,6 +18,7 @@
 		"@angular/router": "^8.2.8",
 		"@ckeditor/ckeditor5-angular": "^1.2.3",
 		"asdasd123qwe": "0.0.3",
+		"mongoose": "^4.3.3",
```

Comment:
mongoose is a back-end library that uses MongoDB and you don't need it on the front-end